### PR TITLE
SASS-553 - Copied personal-income-fe directory structure into the app

### PR DESCRIPTION
**Pull request by Dula-J**

### CODE REVIEW

File: app/forms/PriorOrNewAmountForm.scala [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+package forms
+
+import models.formatHelpers.PriorOrNewAmountModel
+import play.api.data.Forms._
+import play.api.data.format.Formatter
+import play.api.data.{Form, FormError}
+import play.api.i18n.Messages
+
+object PriorOrNewAmountForm {
```
Comment (BenCosfordHMRC) [inline app/forms/PriorOrNewAmountForm.scala:25]: I think this form can be removed as we are removing it from the current FE.

File: app/config/ConfigKeys.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+object ConfigKeys {
+  val signInUrl = "signIn.url"
+  val signInContinueBaseUrl = "signIn.continueBaseUrl"
+  val dividendsBaseUrl = "microservice.dividends.url"
```
Comment (BenCosfordHMRC) [inline app/config/ConfigKeys.scala:22]: can we remove dividendsBaseUrl?

File: app/models/formatHelpers/PriorOrNewAmountModel.scala [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.formatHelpers
+
+case class PriorOrNewAmountModel(whichAmount: String, amount: Option[BigDecimal])
```
Comment (BenCosfordHMRC) [inline app/models/formatHelpers/PriorOrNewAmountModel.scala:19]: Think this can be removed

File: conf/application.conf [hunk 4]
```diff
@@ -19,26 +33,42 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModu
...
+# The secret key is used to secure cryptographics functions.
+# If you deploy your application to several instances be sure to use the same key!
+# Not set here so that MDTP frontends share the same secret key in the local environment
+# (see common.conf in frontend-bootstrap).
+# In server environments the secret comes from app-config-common
+# play.http.secret.key="NWCsJFOz4nRxQbYsUjUgzCzsiT5s9KNfiCfUDjWMDvNqCgHBSzNYwI8P7huPYGCa"
 
 microservice {
+
+  url = "http://localhost:9317"
```
Comment (BenCosfordHMRC) [inline conf/application.conf:64]: just checking - is this port correct?

File: conf/application.conf [hunk 5]
```diff
@@ -19,26 +33,42 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModu
...
+
   metrics {
     graphite {
       host = localhost
       port = 2003
       prefix = play.${appName}.
-      enabled = true
+      enabled = false
```
Comment (BenCosfordHMRC) [inline conf/application.conf:71]: should be true by default

File: conf/application.conf [hunk 6]
```diff
@@ -48,6 +78,44 @@ microservice {
       host = localhost
       port = 9250
     }
+
+    income-tax-dividends {
```
Comment (BenCosfordHMRC) [inline conf/application.conf:82]: dividends & interest can be removed, add in the employment BE

File: project/plugins.sbt [hunk 7]
```diff
@@ -1,9 +1,24 @@
...
+
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
+
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
+
+addSbtPlugin("net.ground5hark.sbt" % "sbt-concat" % "0.2.0")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-uglify" % "2.0.0")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.4")
```
Comment (BenCosfordHMRC) [inline project/plugins.sbt:24]: would these versions be okay to have?
addSbtPlugin("com.typesafe.play" % "sbt-plugin"         % "2.7.9")
addSbtPlugin("org.irundaia.sbt"  % "sbt-sassify"        % "1.5.1")

**Review by BenCosfordHMRC — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 4 above)
Comment (Dula-J) [reply conf/application.conf:64]: Yes

File: conf/application.conf (hunk 5 above)
Comment (Dula-J) [reply conf/application.conf]: Changed, must of forgot to revert it after running tests

File: project/plugins.sbt (hunk 7 above)
Comment (Dula-J) [reply project/plugins.sbt:24]: Done

File: app/models/httpResponses/ErrorResponse.scala [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.httpResponses
+
+import play.api.libs.json.{Json, OWrites}
+
+case class ErrorResponse(status: Int, message: String)
```
Comment (BenCosfordHMRC) [inline app/models/httpResponses/ErrorResponse.scala:21]: must have missed this earlier, I don't think we need this file

**Review by BenCosfordHMRC — CHANGES_REQUESTED** [review]

**Review by BenCosfordHMRC — APPROVED** [review]

**Pull request merged**